<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Ramyas-MacBook-Air.local" failures="1" tests="5" name="TestForLoginPage_firefox" time="30.54" errors="0" timestamp="2022-05-20T23:24:09 CDT">
  <testcase classname="com.salesforce.tests.LoginPageTestscripts" name="CheckRemeberMe3" time="8.974"/>
  <testcase classname="com.salesforce.tests.LoginPageTestscripts" name="ForgotPassword4A" time="0.91"/>
  <testcase classname="com.salesforce.tests.LoginPageTestscripts" name="ForgotPassword4B" time="0.311"/>
  <testcase classname="com.salesforce.tests.LoginPageTestscripts" name="LoginErrorMessage1" time="0.541"/>
  <testcase classname="com.salesforce.tests.LoginPageTestscripts" name="LoginToSalesForce2" time="1.981">
    <failure type="java.lang.AssertionError" message="expected [Welcome to your free trial not displayed,Test Failed] but found []">
      <![CDATA[java.lang.AssertionError: expected [Welcome to your free trial not displayed,Test Failed] but found []
at com.salesforce.tests.LoginPageTestscripts.LoginToSalesForce2(LoginPageTestscripts.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- LoginToSalesForce2 -->
</testsuite> <!-- TestForLoginPage_firefox -->
